content/docs/2.4/operate/metrics-server.md

```diff
@@ -74,4 +74,24 @@ During its work, KEDA updates each ScaledObject with some relevant information w
 You can recover the metric names from a ScaledObject using `kubectl`:
 ```bash
  kubectl get scaledobject SCALEDOBJECT_NAME -n NAMESPACE -o jsonpath={.status.externalMetricNames}
+```
+
+## How to get metric if there are more than 1 ScaledObject with same metric name
+
+KEDA will try to select the proper `ScaledObject` for your metric and it should be only one. In case of having more than 1 `ScaledObject` in the same namespace with the same metric name, an error like this will be thrown:
```

Comment:
```suggestion
KEDA will try to select the proper `ScaledObject` for your metric and there should only be one. In case of having multiple `ScaledObject`s in the same namespace with the same metric name, an error like this will be thrown:
```